src/library_pthread.js

```diff
@@ -400,6 +400,28 @@ var LibraryPThread = {
         'workerID': PThread.nextWorkerID++,
 #endif
       });
+    }),
+
+    loadWasmModuleToAllWorkers: function(onMaybeReady) {
+#if !PTHREAD_POOL_SIZE
+      onMaybeReady();
+#else
+      // Instantiation is synchronous in pthreads and we assert on run dependencies.
```

Comment:
```suggestion
      // Instantiation is synchronous in pthreads.
```
(Comment was just copied, and seems to have been stale before, but might as well clean it up now.)